packages/utils/src/ast.ts

```diff
@@ -199,34 +142,25 @@ function getRawTemplateValue(value: string) {
  * - Breaks up tokens that would otherwise result in SES errors, such as HTML
  * comment tags `<!--` and `-->` and `import(n)` statements.
  *
- * @param ast - The AST to post process.
- * @returns The modified AST.
+ * @param code - The code to post process.
+ * @param attachComments - Whether to include comments in the output.
+ * @returns The modified code.
  */
-export function postProcessAST(ast: Node): Node {
-  // Modifies the AST in place.
-  traverse(ast, {
-    enter(path) {
-      const { node } = path;
-
-      COMMENTS_KEYS.forEach((key) => {
-        const comments = node[key];
-        if (!comments) {
-          return;
-        }
-
-        // Break up tokens that could be parsed as HTML comment terminators. The
-        // regular expressions below are written strangely so as to avoid the
-        // appearance of such tokens in our source code. For reference:
-        // https://github.com/endojs/endo/blob/70cc86eb400655e922413b99c38818d7b2e79da0/packages/ses/error-codes/SES_HTML_COMMENT_REJECTED.md
-        comments.forEach((comment) => {
-          comment.value = comment.value
-            .replace(new RegExp(`<!${'--'}`, 'gu'), '< !--')
-            .replace(new RegExp(`${'--'}>`, 'gu'), '-- >')
-            .replace(/import(\(.*\))/gu, 'import\\$1');
-        });
-      });
-    },
-
+export function postProcessAST(code: string, attachComments = true): string {
```

Comment:
Perhaps this should be renamed if it doesn't take an AST anymore.

Reply:
Will merge this with the other `postProcess` function.